packages/contracts/contracts/MultiPriceFeed.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.11;
+
+import "./Interfaces/IPriceFeed.sol";
+import "./Dependencies/SafeMath.sol";
+import "./Dependencies/IgOHM.sol";
+import "./Dependencies/Ownable.sol";
+import "./Dependencies/CheckContract.sol";
+
+contract MultiPriceFeed is Ownable, CheckContract, IPriceFeed {
+    using SafeMath for uint256;
+
+    IPriceFeed public ethUSDFeed;
+    IPriceFeed public ohmETHFeed;
+    IgOHM public gOHM;
+    uint256 public lastPrice;
+
+    function setAddresses(
+        address _ethUSDFeedAddress,
+        address _ohmETHFeedAddress,
+        address _gOHMAddress
+    )
+        external
+        onlyOwner
+    {
+        checkContract(_ethUSDFeedAddress);
+        checkContract(_ohmETHFeedAddress);
+        checkContract(_gOHMAddress);
+
+        ethUSDFeed = IPriceFeed(_ethUSDFeedAddress);
+        ohmETHFeed = IPriceFeed(_ohmETHFeedAddress);
+        gOHM = IgOHM(_gOHMAddress);
+
+        _renounceOwnership();
+    }
+
+    function fetchPrice() external override returns (uint) {
+        uint256 ohmETHPrice = ohmETHFeed.fetchPrice();
+        uint256 ethUSDPrice = ethUSDFeed.fetchPrice();
+
+        require(ohmETHPrice > 0, "PriceFeed: OHM-ETH price is zero");
```

Comment:
Do we need to check `PriceFeed::status()` ?  

It might make sense to return the last good price if either underlying feed has `status != Status.chainlinkWorking`, as this mimics the behavior of the underlying `PriceFeed::fetchPrice()`

If we did that might want to also keep track of the aggregated status within here, as well.

What do you think?

Reply:
My idea here was to let the underlying contracts handle their statuses. `fetchPrice` will always return a result. The multi-price feed then doesn't need to care if the OHM-ETH feed is using Tellor and the ETH-USD feed is using Chainlink. These requires are just here to be extra cautious in case there is a bug in their `PriceFeed` code that will liquidate everybody.

Reply:
Was trying to think thru this scenario:

`ETH-USD = Status.bothOraclesUntrusted`
`OHM-ETH = Status.chainlinkWorking`

In that scenario, we'd take the `PriceFeed::lastGoodPrice` from `ETH-USD`, but use the current price for `OHM-ETH`.

Is that better or worse than just taking the `lastGoodPrice` from each feed?  I'm not really sure as I don't know how old each respective `lastGoodPrice` is, but it seems like it would be less risky than using both an old and a current price? 

I wonder how often both Chainlink and tellor price feeds both go down, maybe I'm overthinking a scenario that won't occur

Reply:
Yeah, I don't really know what the best option is. I wonder if we can see how the ETH-USD feed holds up in liquity. All this complexity could be for nothing. I'll try to dig in

Reply:
Just added some logic to account for these scenarios.  Think the complexity is minor, and makes it safer.  Also added some events

Reply:
I love it. Great stuff